src/utils/lookup-table-utils.ts

```diff
@@ -54,11 +56,16 @@ export const languageMatcherCaseStatement = (languageColumn: string | undefined)
   SUPPORTED_LOCALES.map((locale) => {
     const lang = locale.split('-')[0].toLowerCase();
     const tLang = lang;
-    languageMatcher.push(`WHEN LOWER("${languageColumn}") LIKE '${lang}%' THEN '${locale.toLowerCase()}'`);
+    languageMatcher.push(pgformat('WHEN LOWER(%I) LIKE %L THEN %L', languageColumn, `%${lang}%`, locale.toLowerCase()));
```

Comment:
Are the param placeholders meant to be unique chars? `%L` is repeated here but you have 3 different params.

Reply:
`%L` is a SQL Litteral/Value and `%I` is a SQL Identifier and `%s%` is a raw string.  The library replaces things in the order it's given.  It works in much the same was `printf`

Reply:
👍 should've just read the docs but I was being lazy :D